src/laiOffer/class2_recursion_binarysearch/KCloestInSortedArray.java

```diff
@@ -0,0 +1,48 @@
+package laiOffer.class2_recursion_binarysearch;
+
+/**
+ * Description
+ * Given a target integer T, a non-negative integer K and an integer array A sorted in ascending order, find the K closest numbers to T in A.
+ *
+ * Assumptions
+ *
+ * A is not null
+ * K is guranteed to be >= 0 and K is guranteed to be <= A.length
+ * Return
+ *
+ * A size K integer array containing the K closest numbers(not indices) in A, sorted in ascending order by the difference between the number and T.
+ * Examples
+ *
+ * A = {1, 2, 3}, T = 2, K = 3, return {2, 1, 3} or {2, 3, 1}
+ * A = {1, 4, 6, 8}, T = 3, K = 3, return {4, 1, 6}
+ */
+
+public class KCloestInSortedArray {
+    public int[] kClosest(int[] array, int target, int k) {
+       if (array == null || array.length == 0) return new int[]{};
+       int left = 0, right = array.length - 1;
+       while (left < right - 1) {
+           int mid = left + (right - left)/2;
+           if (array[mid] <= target) {
```

Comment:
`if (array[mid] == target)` why just `left = mid`? in that way, mid is exactly or closed index to target number